src/utils/json_utils.py

```diff
@@ -13,6 +13,23 @@ def write(path, data):
         json.dump(data, outfile, indent=4)
 
 
+# Remove keys from a dictionary at the specified depth
+# Depth 1 removes keys from the first level of the dictionary
+# Depth N removes keys from the first N levels of the dictionary
+def remove_keys(data, keys_to_remove, depths_to_search=1):
```

Comment:
similar to my first comment, document the function such that the intellisense will show it along with the function

Reply:
complete